ush/templates/FV3LAM_wflow.xml

```diff
@@ -719,6 +760,7 @@ MODULES_RUN_TASK_FP script.
 {%- endif %}
 
 
+{%- if not do_ensemble %}
```

Comment:
similarly, we can use "{%- if do_nonvar_cldanal or do_refl2tten %}" to turn on/off this task.  
{%- if not do_ensemble %} is not required.